app/templates/base.html

```diff
@@ -18,8 +18,14 @@
 
 {% block header %}
   {{ govukHeader({
-    'homepageUrl': url_for('main.upload'),
-    'serviceUrl': url_for('main.upload'),
+    'homepageUrl': url_for('main.index'),
```

Comment:
Are we adding a seperate index page or renaming the current upload.html to index.html?

Reply:
The index route currently redirects to the upload page